### DISCUSSION TITLE
Only evil can write such program to get more popularity ... - no ethic

### DISCUSSION THREAD

**Discussion by ChameleonRed** [RonSijm/ButtFish#35] [General]
I do not know if Niemann cheat or not so I will not do sin of blaming other. For me it is clear that there is not evidence.

Whatever what some players is doing is wrong ... whatever coding such program to get fame and propagate such evil ideas is much worse - it is like Hitler - he do not kill anybody but his people order to kill millions - he enabled killing of people.

Same wrong is with your program you enable wrong things without reason - no ethic

**Comment by ChameleonRed** [comment]
You enable not cheating for much more people what is terrible ... - you should delete this from github to stop this madness.

**Comment by RonSijm** [comment]
lmfao

**Comment by robcxyz** [comment]
Damn you got compared to Hitler with your butt project. Is this best left for discussion?

**Reply by RonSijm** [reply]
> You should leave this issue up for discussion as I think this needs a proper roasting.

Alright, fine with me... it was open still for discussion- but it's not an issue, so migrated this to a discussion topic.

Go discuss how I'm literally Hitler (or not) - Or roast this guy.. whatever

**Reply by GeorgeWL** [reply]
Better to just report and block the guy tbh

**Comment by GeorgeWL** [comment]
> I do not know if Niemann cheat or not so I will not do sin of blaming other. For me it is clear that there is not evidence.
> 
> Whatever what some players is doing is wrong ... whatever coding such program to get fame and propagate such evil ideas is much worse - it is like Hitler - he do not kill anybody but his people order to kill millions - he enabled killing of people.
> 
> Same wrong is with your program you enable wrong things without reason - no ethic
> 

You're an absolute and complete idiot. Blocked and Reported
